geode-core/src/main/java/org/apache/geode/redis/GeodeRedisServer.java

```diff
@@ -424,14 +429,26 @@ private void initializeRedis() {
       RegionFactory<ByteArrayWrapper, ByteArrayWrapper> rfString = cache.createRegionFactory(DEFAULT_REGION_TYPE);
       RegionFactory<ByteArrayWrapper, HyperLogLogPlus> rfHLL = cache.createRegionFactory(DEFAULT_REGION_TYPE);
       Region<ByteArrayWrapper, ByteArrayWrapper> stringsRegion;
-      if ((stringsRegion = this.cache.getRegion(STRING_REGION)) == null)
-        stringsRegion = rfString.create(GeodeRedisServer.STRING_REGION);
+      InternalRegionArguments ira = new InternalRegionArguments().setInternalRegion(true);
+      AttributesFactory af = new AttributesFactory();
+      af.setScope(Scope.LOCAL);
+      RegionAttributes ra = af.create();
       Region<ByteArrayWrapper, HyperLogLogPlus> hLLRegion;
-      if ((hLLRegion = this.cache.getRegion(HLL_REGION)) == null)
-        hLLRegion = rfHLL.create(HLL_REGION);
       Region<String, RedisDataType> redisMetaData;
-      if ((redisMetaData = this.cache.getRegion(REDIS_META_DATA_REGION)) == null)
-        redisMetaData = rfMeta.create(REDIS_META_DATA_REGION);
+      try {
+        if ((stringsRegion = this.cache.getRegion(STRING_REGION)) == null)
+          stringsRegion = rfString.create(GeodeRedisServer.STRING_REGION, ra, ira);
+        if ((hLLRegion = this.cache.getRegion(HLL_REGION)) == null)
+          hLLRegion = rfHLL.create(HLL_REGION, ra, ira);
+        if ((redisMetaData = this.cache.getRegion(REDIS_META_DATA_REGION)) == null)
+          redisMetaData = rfMeta.create(REDIS_META_DATA_REGION, ra, ira);
+      }
+      catch (IOException | ClassNotFoundException e) {
```

Comment:
The addition of this try/catch does not look related to region names. Seems like it should have its own ticket.